package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@google-cloud/firestore",
   "description": "Firestore Client Library for Node.js",
-  "version": "0.15.0",
+  "version": "0.15.2",
```

Comment:
Your PR description says 15.0.2.  Please make sure the final commit message says the right thing. :-)